src/client/trackers/realUserMonitoringForPerformance.js

```diff
@@ -95,8 +94,9 @@ export const realUserMonitoringForPerformance = ({ sampleRate } = {}) => {
 		}
 	});
 
-	new Perfume({
-		analyticsTracker,
-		logging: false
-	});
+	onCLS(recordMetric, { reportAllChanges: true });
+	onFCP(recordMetric);
+	onFID(recordMetric);
+	onLCP(recordMetric);
```

Comment:
**question**: do we want to set reportAllChanges to true in onLCP as well, so that recordMetric gets called whenever a new largest-contentful-paint performance entry is dispatched?

Reply:
We don't, because we currently only send the first set of recorded metrics. It seems like `onCLS` will _never_ file if we don't include `reportAllChanges` whereas `onLCP` will. So this isn't about making sure we sent all tracking updates, but making sure we have the data to send our single initial tracking event.

(In future we may want to send changes though).